### PR TITLE
bug fix with metrics and logs

### PR DESCRIPTION
**Pull request by aneeshkp**
Fixed bugs:
GM Metrics
GM Clock Type

### CODE REVIEW

File: pkg/daemon/daemon.go [hunk 1]
```diff
@@ -588,26 +586,33 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool) {
...
-							CfgName:     p.configName,
-							IFace:       iface,
-							Values: map[event.ValueType]int64{
-								event.OFFSET: int64(ts2phcsOffset),
-							},
-							ClockType:  clockType,
-							Time:       time.Now().Unix(),
-							WriteToLog: true,
-							Reset:      false,
+						updateClockStateMetrics(p.name, iface, FREERUN)
```
Comment (jzding) [inline pkg/daemon/daemon.go:593]: this line should move to line 599 in an `else` statement of checking Max/MinOffsetThreshold

File: pkg/ublox/ublox.go [hunk 2]
```diff
@@ -31,7 +31,7 @@ const (
 	// CMD_PROTO_VERSION ...
 	CMD_PROTO_VERSION = " -p MON-VER"
 	// CMD_VOLTAGE_CONTROLER ...
-	CMD_VOLTAGE_CONTROLER = " -v 1 -z CFG-HW-ANT_CFG_VOLTCTRL %d"
+	CMD_VOLTAGE_CONTROLER = " -v 1 -z CFG-HW-ANT_CFG_VOLTCTRL, %d"
```
Comment (jzding) [inline pkg/ublox/ublox.go:34]: This won't work. There should not be a space after `,`.

Comment (josephdrichard) [reply pkg/ublox/ublox.go:34]: This can go in the ptpconfig?

File: pkg/daemon/daemon.go (hunk 1 above)
Comment (aneeshkp) [reply pkg/daemon/daemon.go:593]: since we are bringing in max/min threshold values,They should be applied for all configurations

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**